docs/translations.md

```diff
@@ -15,3 +15,5 @@ If there's not, then today is your day to lead this effort! Here's how to start:
 1. Create a new directory in `_articles/` for your language e.g. `_articles/es/`, copy each guide from `_articles/` into that folder and translate each guide.
 1. Run `script/test` and make sure there are no failures with your translation files. Note that you may need to rename anchor links or leave some headers in English to fix broken links.
 1. Send a pull request.
+
+Completing an initial translation of the whole site is a fairly large task. One way to break that task up is to work with other translators through pull requests on your fork. Example: [pull requests on fork for German translation](https://github.com/katrinleinweber/opensource.guide/pulls?q=is%3Apr+is%3Aclosed) and corresponding [initial pull request for German translation](https://github.com/github/opensource.guide/pull/577) on this repository.
```

Comment:
Can we also list https://github.com/open-craft-guild/open-source-guide as Ukrainian translation home?